### ISSUE TITLE
The bots are missing from the developer guides

### ISSUE THREAD

**Issue by supakeen** [osbuild/osbuild.github.io#82]
We have many cool bots, they should be listed in the developer guides so other projects can benefit from our setup.

**Comment by ochosi** [comment]
Indeed... The problem is that some of them (centos-bot, app-interface-bot, guides-bot) _have to_ live behind the VPN, so publicly documenting them would feel odd.

We could ofc document the ones we have on GitHub, but that would at this point mean duplicating parts of our internal guides.
